cdk/lib/monitoring.ts

```diff
@@ -71,7 +71,7 @@ export class Monitoring extends GuStack {
 		});
 
 		const monitoringDuration: Duration =
-			stage === 'PROD' ? Duration.minutes(2) : Duration.days(1); // Every day for CODE; Every 2 minutes for PROD.
+			stage === 'PROD' ? Duration.minutes(2) : Duration.minutes(2); // Every day for CODE; Every 2 minutes for PROD.
```

Comment:
Presumably this is for testing? Does it need to be switched back to a lower duration before merge?

Reply:
Yes, it is only for testing and will be reverted before prod